### PR TITLE
Generalise

### PR DESCRIPTION
**Pull request by rhattersley**

### CODE REVIEW

**Review by KatherineTomkins — APPROVED** [review]
Small observation, but where we  say "how to download data..." should this be "How to download Met Office data..."? to avoid sounding a bit too generalised? As in, we are trying to show how to use the MO products, not show them how to download any data from anywhere?

**Comment by rhattersley** [comment]
> a bit too generalised?

👍 I wondered about that too, but wanted to get away from "Met Office data" as it isn't limited to that, e.g. we may expose data from other providers. How about something like "data from Met Office object services"?

**Comment by rhattersley** [comment]
@KatherineTomkins - I've attempted to clarify things along the lines described above. If you're happy with the result, please merge this PR. Otherwise, just let me know any other snags. Thanks!

File: aws-example/README.md [hunk 1]
```diff
@@ -128,29 +128,29 @@ Statement:
...
-   'pressure' returns 'surface_air_pressure' and 'humidity' returns 'relative_humidity'.
-   All diagnostic name strings other than 'temperature', 'pressure' and 'humidity' are ignored.
-   
-   * The -v flag is set for verbose output.
-   
-   * The -k flag set keeps messages until deleted by the user.
+   * Clone this repository and ensure it is the current directory.
+
+   * To begin polling, execute the `download.py` script with the name of
+     your SQS queue and your API key as arguments.
```
Comment (adrianmilne) [inline aws-example/README.md:140]: Should the order of these be reversed for clarity? - i.e. `$ python download.py <api_key> <queue-name>`

Comment (rhattersley) [reply aws-example/README.md:140]: The subsequent example is out of date. I'll update... 👍

**Pull request merged**